tests/handelingWithAlert.spec.js

```diff
@@ -0,0 +1,117 @@
+const { test, expect } = require('@playwright/test');
+
+
+
+test.describe('Handling alerts/modal in web application', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('https://demoqa.com/alerts');
```

Comment:
need to navigate the application once then start testing but here you navigate the same application again and again. So what you have to do just navigate once page'https://demoqa.com/alerts' then go back and forth to test using formula.

Reply:
Can you please explained in with a screenshot please.